.pre-commit-config.yaml

```diff
@@ -10,6 +10,11 @@ repos:
     rev: 24.4.2
     hooks:
     -   id: black
```

Comment:
We should remove black if we have ruff now.

Reply:
Actually, I think the ruff linter is orthogonal to `black` as a formatter. There is also the ruff formatter (which is a Black alternative) but it is currently disabled. Using the ruff linter together with Black is a [common use case](https://docs.astral.sh/ruff/faq/#is-the-ruff-linter-compatible-with-black).

Open to switch to the ruff formatter in a separate PR if you think that makes sense - but for now, I propose to stick with Black.

Reply:
agreed on using both ruff and black

Reply:
Actually I am confused now. Do we still get autofixes from ruff if we don't use it as a formatter?

Reply:
Yes, we do.